src/BlocHost-Morphic/BlMorphicSpaceHostMorph.class.st

```diff
@@ -23,7 +23,12 @@ BlMorphicSpaceHostMorph >> drawOn: aCanvas [
 		ifTrue: [ ^ self ].
 	hostSpace ifNotNil: [ 
 		aCanvas clipBy: self fullBounds during: [ :aClippedCanvas |
-			aClippedCanvas drawImage: spaceForm at: self position ] ]
+			SystemVersion current major >= 12 ifTrue: [
+				| formSet |
+				formSet := (Smalltalk at: #FormSet) extent: self extent depth: spaceForm depth forms: { spaceForm }.
```

Comment:
Was `FormSet` introduced in Pharo 12?

In any case, I have doubts about `Smalltalk at: #FormSet`. I think Pharo moves towards moving to use `self class environment classNamed: #FormSet` instead of it. @tesonep should now

Reply:
I introduced FormSet in Pharo 12 in [Pharo pull request #14998](https://github.com/pharo-project/pharo/pull/14998).

It seemed better to keep the implementation the same as in the ‘dev’ branch. I don’t know whether the ‘dev’ branch is supposed to still be usable with Pharo 11 or even older versions though. If not, I could drop the SystemVersion checks and the use of Smalltalk in the ‘dev’ branch first and update this pull request accordingly.

Reply:
I have opened pull request #544 to remove support for Pharo 11.

Reply:
Many thanks for your work @Rinzwind, and I apologise for not answering fast to your comment and PR. I talked with @tesonep and, as @plantec wrote in the PR, we will keep support for P11. I will check how to merge your contribution, because it's great and we want it.